ludwig/models/predictor.py

```diff
@@ -388,6 +390,13 @@ def batch_evaluation(self, dataset, collect_predictions=False, collect_logits=Fa
                     outputs = self._predict_on_inputs((inputs, targets))
                     preds = self.model.outputs_to_predictions(outputs)
 
+                    for key in inputs:
+                        example_inputs[key].extend(inputs[key])
+                    for key in targets:
+                        example_targets[key].extend(targets[key])
+                    for key in preds:
+                        example_outputs[key].extend(preds[key]["predictions"])
```

Comment:
Is there a `response` key from the predictions dictionary? I remember adding it for computing decoded-text metrics.

Reply:
In `preds[key]`, I only see "predictions", "probabilities", and "logits"